src/input.yaml

```diff
@@ -0,0 +1,18 @@
+---
+#rules
+allow-90-degree-rotations: true
+allow_180-degree-rotations: false
+
+#image location
+im_location: "./images"
```

Comment:
We need to create an image dir

Reply:
My thought was the user would provide their own image dir, and here would provide the path to it (we will need to make one for testing)